Cython/Compiler/PyrexTypes.py

```diff
@@ -4173,7 +4173,7 @@ def declaration_code(self, entity_code,
             elif self.typedef_flag:
                 base_code = self.cname
             else:
-                base_code = "enum %s" % self.cname
+                base_code = "__PYX_ENUM_DECL %s" % self.cname
```

Comment:
Does this really apply to plain C enums as well? It seemed to me that it's only a C++ enum issue. I would like to keep the code impact of this as low as possible.

Reply:
It doesn't have to, so I renamed the macro `__PYX_ENUM_CLASS_DECL` and now use it only for the declaration of C++ enums.